src/jit/importer.cpp

```diff
@@ -6462,6 +6463,11 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         call->gtCall.callSig  = new (this, CMK_CorSig) CORINFO_SIG_INFO;
         *call->gtCall.callSig = calliSig;
 #endif // DEBUG
+
+        if (IsTargetAbi(CORINFO_CORERT_ABI))
```

Comment:
We'll also need this transform for the GVM call, and in that case, the opcode will be a CALL_VIRT. For v1 of the GVM implementation, the result returned from the jit interface getCallInfo will have a CORINFO_VIRTUALCALL_LDVIRTFTN callinfo kind. The codegen will contain a lookup into a R2R cell, which will return the GVM calli target (which could be a fat function pointer)